testing/tests/i18n.rs

```diff
@@ -0,0 +1,91 @@
+#![cfg(feature = "localization")]
+
+use askama::{langid, Locale, Template};
+
+askama::localization!(LOCALES);
+
+/*
+#[derive(Template)]
```

Comment:
What's the deal with the commented out tests here? Do we still need them?

Reply:
Ah, my bad forgot to remove them, they were tests for the original implementation(#434). I will remove them once I have time.